CMakeLists.txt

```diff
@@ -109,4 +109,4 @@ endif()
 
 if(BUILD_WITH_PACKAGE)
     include(project/CPackConfig)
-endif()
+endif()
```

Comment:
Unneeded change. Please add new line at the end of file and configure your IDE to always add a new line at the end of file. In VSCode it can be done by settings `"editor.formatOnSave": true` (or similar property).